documentation/book/assembly-security.adoc

```diff
@@ -0,0 +1,26 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='security-{context}']
+= Security
+
+{ProductName} supports encrypted communication between the Kafka and {ProductName} components using the TLS protocol.
+Communication between Kafka brokers (interbroker communication), between Zookeeper nodes (internodal communication), and between these and the {ProductName} operators is always encrypted.
+Communication between Kafka clients and Kafka brokers will be encrypted according how the cluster is configured.
+For the Kafka and {ProductName} components, TLS certificates are also used for authentication.
+
+The Cluster Operator sets up the TLS certificates to provide this encryption and authentication.
```

Comment:
Last sentence - Use the Cluster Operator to set up the TLS certificates, which enable encryption and authentication [in your cluster]. 

Not sure about the scope of the last part of the sentence in brackets.

Reply:
@sjay72 the CO does this automatically without any action on the user's part, at least for internal communication (the user can't even disable it). For communication between brokers and clients it's optional. Your rewording is somewhat misleading because it suggests the user needs to do something, and that's only true if they want to use TLS for client communication, and even then they'd have to configure plaintext communication in basically the same way, so there's no more _use of_ the CO to have TLS than there is for the alternative.

A more accurate wording would be something like:

The Cluster Operator automatically sets up TLS certificates to enable encryption and authentication within your cluster, and sets up other TLS certificates if you want to enable encryption or TLS authentication between Kafka brokers and clients.

But this seems a little long. Do you have a better suggestion?

Reply:
Oh! That's convenient. I would just make it 2 sentences:
The Cluster Operator automatically sets up TLS certificates to enable encryption and authentication within your cluster. It also sets up other TLS certificates if you want to enable encryption or TLS authentication between Kafka brokers and clients.